tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -562,6 +561,8 @@ def feature_importance(self, dataset=None, model=None, features=None, feature_st
         Note that calculating feature importance can be a very computationally expensive process, particularly if the model uses hundreds or thousands of features. In many cases, this can take longer than the original model training.
         To estimate how long `feature_importance(model, dataset, features)` will take, it is roughly the time taken by `predict_proba(dataset, model)` multiplied by the number of features.
 
+        Note: For highly accurate stddev and z_score estimates, it is recommend to set `subsample_size` to at least 5,000 if possible and `num_shuffle_sets` to at least 10.
+
```

Comment:
Can docs have a link to PFI score tutorial like this: 
https://explained.ai/rf-importance/

Folks keep asking what the scores represent.